specification/cognitiveservices/data-plane/Speech/BatchAvatar/preview/2024-04-15-preview/batchavatar.json

```diff
@@ -0,0 +1,846 @@
+      "required": [
+        "voice"
+      ],
+      "type": "object",
+      "properties": {
+        "voice": {
+          "description": "Voice name for PlainText input. You can find platform voice list here https://learn.microsoft.com/azure/ai-services/speech-service/language-support?tabs=tts.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "style": {
+          "description": "Style name for PlainText input.",
+          "type": "string"
+        },
+        "rate": {
+          "description": "Speaking rate for PlainText input. For supported values, see https://learn.microsoft.com/azure/ai-services/speech-service/speech-synthesis-markup-voice#adjust-prosody.",
+          "type": "string"
+        },
+        "pitch": {
+          "description": "Pitch for PlainText input. For supported values, see https://learn.microsoft.com/azure/ai-services/speech-service/speech-synthesis-markup-voice#adjust-prosody.",
+          "type": "string"
+        },
+        "volume": {
+          "description": "Volume for PlainText input. For supported values, see https://learn.microsoft.com/azure/ai-services/speech-service/speech-synthesis-markup-voice#adjust-prosody.",
+          "type": "string"
+        }
+      }
+    },
+    "BatchSynthesisError": {
+      "description": "Batch Synthesis Error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Top level error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message",
+          "type": "string"
+        }
+      }
+    },
+    "BatchSynthesisInput": {
+      "description": "Batch Synthesis input.",
+      "required": [
+        "content"
+      ],
+      "type": "object",
+      "properties": {
+        "content": {
+          "description": "Input content. It could be SSML or plain text.",
+          "minLength": 1,
+          "type": "string"
+        }
+      }
+    },
+    "BatchSynthesisOutputs": {
+      "description": "Batch Synthesis output.",
+      "type": "object",
+      "properties": {
+        "result": {
+          "description": "sas url of the result file. It would be relative path when \"destinationContainerUrl\" is specified.",
+          "type": "string"
+        },
+        "summary": {
+          "description": "Job summary file path will be provided here.",
+          "type": "string"
+        }
+      }
+    },
+    "BatchSynthesisProperties": {
+      "description": "Detailed properties of batch synthesis task.",
+      "type": "object",
+      "properties": {
+        "timeToLiveInHours": {
+          "description": "How long the batch task will be kept in the system after it has completed. Once the batch task reaches the time to live after completion (succeeded or failed) it will be automatically deleted. The longest supported duration is 744 hours (31 days). The default value is 744 hours. ",
```

Comment:
In additional to decsription, default value better to be put to e.g. `"default": 774`

Reply:
Thanks! Added `"default"`